### PR TITLE
Bump Connect to version 2024.01.0

### PR DESCRIPTION
**Pull request by Lytol**
This PR was created automatically by the Posit Connect release scripts.

### CODE REVIEW

**Review by shepherdjerred — APPROVED** [review]

File: .github/workflows/build-prerelease.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
           - {product: "workbench", type: "preview", os: "ubuntu2204", r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.17", py-alternate: "3.8.17"}
           - {product: "connect", type: "daily", os: "ubuntu2204", r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.17", py-alternate: "3.8.17"}
           - {product: "connect-content-init", type: "daily", os: "ubuntu2204", r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.17", py-alternate: "3.8.17"}
-          - {product: "package-manager", type: "daily", os: "ubuntu2204", r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.17", py-alternate: "3.8.17"}
+          - {product: "package-manager", type: "daily", os: "ubuntu2204", r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.12.1", py-alternate: "3.11.7"}
```
Comment (aronatkins) [inline .github/workflows/build-prerelease.yaml:33]: I don't understand why the Connect changes are adjusting the package-manager data.

File: .github/workflows/build-release.yaml [hunk 2]
```diff
@@ -30,6 +30,7 @@ jobs:
           - {os: 'ubuntu2204', r-primary: "4.2.0", r-alternate: "3.6.2", py-primary: "3.9.5", py-alternate: "3.8.10"}
           - {os: 'ubuntu2204', r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.14", py-alternate: "3.8.15"}
           - {os: 'ubuntu2204', r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.9.17", py-alternate: "3.8.17"}
+          - {os: 'ubuntu2204', r-primary: "4.2.3", r-alternate: "4.1.3", py-primary: "3.12.1", py-alternate: "3.11.7"}
```
Comment (aronatkins) [inline .github/workflows/build-release.yaml:33]: related: The Connect changes should not be modifying this information.

File: package-manager/Dockerfile.ubuntu2204 [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ARG R_VERSION=4.2.3
 ARG R_VERSION_ALT=4.1.3
-ARG PYTHON_VERSION=3.9.17
-ARG PYTHON_VERSION_ALT=3.8.17
+ARG PYTHON_VERSION=3.12.1
+ARG PYTHON_VERSION_ALT=3.11.7
```
Comment (aronatkins) [inline package-manager/Dockerfile.ubuntu2204:4]: Why are the Package Manager versions changing in this PR?

**Review by aronatkins — CHANGES_REQUESTED** [review]
The intent is for these to be Connect-only changes, right?

**Comment by Lytol** [comment]
As everyone all noticed, this PR is wonky so I'll be closing and re-opening.  I'm investigating root causes on our release script to see why this is happening.

**Pull request closed without merging**